confounds/visualize.py

```diff
@@ -23,8 +24,7 @@ def samplets(features,
 
 
 def pairs(confounds,
```

Comment:
Hi @jameschapman19 are you able to share few example visualizations to add to our docs website? esp for a differing number of variables (n>3)..